src/windows/volume.cc

```diff
@@ -65,6 +65,41 @@ HRESULT drivelist::volume::GetReadOnlyFlag(const wchar_t letter, BOOL *out) {
   return result;
 }
 
+HRESULT drivelist::volume::HasFileSystem(const wchar_t letter, BOOL *out) {
```

Comment:
> Consider creating a FAT32 partition on a disk (which updates the MBR) but then not formatting it. The volume has a valid partition type, but it doesn't have any filesystem yet...

Oh yeah, I didn't think of that, and I can indeed trigger a bug in such scenario. How should we check if the partition is indeed there though?